examples/tuple_types/test/tests.ts

```diff
@@ -495,6 +495,26 @@ export function get_tests(
                     ok: result[0].hello === 0n && result[1].goodbye === 1n
                 };
             }
+        },
+        {
+            name: 'nested tuple test',
+            test: async () => {
+                let expected_result: [[string, [number, number]], bigint] = [
```

Comment:
Is there a reason this is let instead of const?

Reply:
nope. Good catch :)

Reply:
nope. good catch!